src/Symfony/Component/Intl/CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
 CHANGELOG
 =========
 
+7.1
+---
+
+* Transfert all emoji code & data to a new dedicated component: `Symfony\Component\IntlEmoji`
+* Deprecate `EmojiTransliterator` in favor of `Symfony\Component\IntlEmoji\EmojiTransliterator`
```

Comment:
```suggestion
 * Transferred all emoji code & data to a new dedicated component: `Symfony\Component\IntlEmoji`
 * Deprecate `EmojiTransliterator` in favor of `Symfony\Component\IntlEmoji\EmojiTransliterator`
```

Reply:
Why `Symfony\Component\IntlEmoji` and not `Symfony\Component\Emoji` ?

Reply:
My (humble personal) opinion on this would be:
* Symfony\Component\Emoji could be something to manipulate... emojis (so unicode caracters, encoding, ranges)  but it could then probably be into the String component
* Symfony\Component\Intl-Emoji is the "localisation-layer" to get every emojis short name, description and variations in any language (something that could be accessible via the current provided files, but probably sooner than later via OS api, php extensions, ...)

In a ideal world, String component could handle emoji transliteration not knowing where the data is from.. and IntlEmoji installed only by who needs exhaustive data and/or in a "polyfill-ish" way.